project/src/main/webapp/api-key.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
delete this file since it's sensitive info

Reply:
sounds good